mcux/middleware/usb/device/usb_device_lpcip3511.c

```diff
@@ -0,0 +1,2335 @@
+/*
+ * Copyright (c) 2016, Freescale Semiconductor, Inc.
+ * Copyright 2016 - 2017,2019 NXP
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include "usb_dc_mcux.h"
```

Comment:
Please update the commit message to specify which version of the SDK the LPC USB driver comes from

Reply:
Done